UnityProject/Assets/Scripts/Core/Database/ApiServer.cs

```diff
@@ -88,6 +88,8 @@ private static async Task<ApiResult<T>> Send<T>(HttpRequestMessage request) wher
 			request.Headers.Add("Accept", "application/json");
 			HttpResponseMessage response = await SecureStuff.SafeHttpRequest.SendAsync(request);
 			string responseBody = await response.Content.ReadAsStringAsync();
+			LoadManager.DoInMainThread(() => Loggy.Log($"{request}"));
```

Comment:
you left these. If you think they are still useful, maybe make them ``Loggy.Debug`` level instead so they don't saturate logs when not wanted.

Reply:
There's no function called Debug
I could change the category

Reply:
what functions are there in Loggy? There should be one for each log level

Reply:
changed it to the API catagory

Reply:
> what functions are there in Loggy? There should be one for each log level

Log, LogWarn, LogError

Reply:
> Log, LogWarn, LogError

BOOOOOOOOD

Reply:
![image](https://github.com/user-attachments/assets/01e9bbfb-0dd4-4ec7-9e3f-2877be23dbc7)

Reply:
ah, use trace then

Reply:
updated to use trace